src/app/zap-templates/zcl/data-model/chip/device-energy-management-cluster.xml

```diff
@@ -0,0 +1,196 @@
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="Energy Management"/>
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0098"/>
+    <field name="PowerAdjustment" mask="0x1"/>
+    <field name="PowerForecastReporting" mask="0x2"/>
+    <field name="StateForecastReporting" mask="0x4"/>
+    <field name="ForecastAdjustment" mask="0x8"/>
+  </bitmap>
+  <cluster apiMaturity="provisional">
+    <name>Device Energy Management</name>
+    <domain>Energy Management</domain>
+    <code>0x0098</code>
+    <define>DEVICE_ENERGY_MANAGEMENT_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster allows a client to manage the power draw of a device. An example of such a client could be an Energy Management System (EMS) which controls an Energy Smart Appliance (ESA).</description>
+    <!--Attributes-->
+    <attribute code="0x0000" side="server" type="ESATypeEnum" define="DEVICE_ENERGY_MANAGEMENT_ESA_TYPE" writable="false" optional="false">ESAType</attribute>
+    <attribute code="0x0001" side="server" type="boolean" define="DEVICE_ENERGY_MANAGEMENT_ESA_CAN_GENERATE" writable="false" optional="false">ESACanGenerate</attribute>
+    <attribute code="0x0002" side="server" type="ESAStateEnum" define="DEVICE_ENERGY_MANAGEMENT_ESA_STATE" default="0" writable="false" optional="false">ESAState</attribute>
+    <attribute code="0x0003" side="server" type="int64s" define="DEVICE_ENERGY_MANAGEMENT_ABS_MIN_POWER" default="0" writable="false" optional="false">AbsMinPower</attribute>
+    <attribute code="0x0004" side="server" type="int64s" define="DEVICE_ENERGY_MANAGEMENT_ABS_MAX_POWER" default="0" writable="false" optional="false">AbsMaxPower</attribute>
+    <!--Conformance feature PA - for now optional-->
+    <attribute code="0x0005" side="server" type="ARRAY" entryType="PowerAdjustStruct" define="DEVICE_ENERGY_MANAGEMENT_POWER_ADJUSTMENT_CAPABILITY" isNullable="true" writable="false" optional="true">PowerAdjustmentCapability</attribute>
+    <!--Conformance feature PFR \| SFR - for now optional-->
+    <attribute code="0x0006" side="server" type="ForecastStruct" define="DEVICE_ENERGY_MANAGEMENT_FORECAST" isNullable="true" writable="false" optional="true">Forecast</attribute>
+    <command source="client" code="0x0000" name="PowerAdjustRequest" optional="true" apiMaturity="provisional">
+      <access op="invoke" privilege="operate"/>
```

Comment:
As I understand, this is the default access and should be omitted here.

Reply:
Removed